src/javascript/app_2/Modules/Trading/Components/Form/form_layout.jsx

```diff
@@ -4,16 +4,18 @@ import ScreenLarge  from './screen_large.jsx';
 import ScreenSmall  from './screen_small.jsx';
 
 const FormLayout = ({
+    contract_id,
```

Comment:
The property name being passed from `trade.jsx` is [`is_contract_visible`](https://github.com/binary-com/binary-static/pull/4198/files#diff-3e1a8b3677c45e7e5bf9a8c546f04a1aR42). It never receives a property named `contract_id`, hence it's always `undefined`.